scripts/node-entrypoint.sh

```diff
@@ -27,5 +29,22 @@ psql -v ON_ERROR_STOP=1 \
 CREATE EXTENSION IF NOT EXISTS btree_gist;
 EOF
 
-# run the main node
-env node /vendor/subql-cosmos/packages/node/bin/run "$@"
\ No newline at end of file
+if [ "$WATCH" = "true" ]
+then
+  # call the first command if WATCH is true
+  echo "WATCH is true. Running the application with nodemon..."
+  rm -rf /dist
+  update_project
+  env yarn exec nodemon --config nodemon.json --exec "NODE_ENV=$NODE_ENV yarn run build && NODE_ENV=$NODE_ENV node /vendor/subql-cosmos/packages/node/bin/run $@"
+else
+  # call the other command if WATCH is not true
+  echo "WATCH is not true. Running the application without nodemon..."
+  # move the dist folder to the mounted folder in run time
+  cp -R /dist/. /app/dist
+  cp -R /types/. /app/src/types
+  cp /project.yaml /app
+  update_project
+  # run the main node
+  NODE_ENV=$NODE_ENV env node /vendor/subql-cosmos/packages/node/bin/run "$@"
```

Comment:
Let's add tini back here. I think something like:
```suggestion
  NODE_ENV=$NODE_ENV /sbin/tini -- env node /vendor/subql-cosmos/packages/node/bin/run "$@"
```

Reply:
`tini` is on the `ENTRYPOINT ["/sbin/tini", "--", "/entrypoint.sh"]` so every command executes inside of `entrypoint.sh` are already running with `tini`

Reply:
Ah, right - thanks for clarifying. :sweat_smile: